app/src/hooks/useSnowbridgeTransferTracker.tsx

```diff
@@ -74,13 +93,11 @@ const useSnowbridgeTransferTracker = () => {
   useEffect(() => {
     ongoingTransfers.forEach(ongoing => {
       if (transfers && 'error' in transfers) return
-      const foundTransfer = transfers.find(transfer =>
-        'extrinsic_hash' in transfer.submitted
-          ? transfer.submitted.extrinsic_hash === ongoing.id
-          : transfer.id === ongoing.id,
-      )
+
+      const foundTransfer = findOngoingTransfer(transfers, ongoing)
+
       if (foundTransfer) {
-        const msg = getStatusMessage(foundTransfer)
+        const msg = getTransferStatus(foundTransfer)
```

Comment:
Should the variable be renamed as well?

```suggestion
        const status = getTransferStatus(foundTransfer)
```